api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -277,7 +282,13 @@ private RdrWorkspace toRdrWorkspace(DbWorkspace dbWorkspace) {
     rdrWorkspace.setStatus(
         org.pmiops.workbench.rdr.model.RdrWorkspace.StatusEnum.fromValue(
             dbWorkspace.getWorkspaceActiveStatusEnum().toString()));
-    rdrWorkspace.setExcludeFromPublicDirectory(false);
+    // For PROD: Confirm workspace details before showing the data on research directory.
+    // As of now Workbench will always exclude workspace details from Public Directory
+    if (workbenchConfigProvider.get().server.shortName.equalsIgnoreCase(PROD_SHORTNAME)) {
```

Comment:
Rather than string matching on the environment name, please create a config flag for this purpose that is true only in prod.